agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3327,7 +3353,12 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             break;
         }
 
-        auto db = AgentDB::get();
+        auto db    = AgentDB::get();
+        auto radio = db->radio(m_fronthaul_iface);
+        if (!radio) {
+            LOG(DEBUG) << "Radio of iface " << m_fronthaul_iface << " does not exist on the db";
```

Comment:
```suggestion
            LOG(DEBUG) << "Radio of interface " << m_fronthaul_iface << " does not exist on the db";
```

same to all occurrences

Reply:
done